cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -13,25 +16,47 @@
                     <td>{{ feature.get_feature_type_display }}</td>
-                    <td>{{ feature.cell_line }}</td>
-                    <td><span>{{ feature.chrom_name }}</span>: <span>{{ feature.location.lower }}-{{ feature.location.upper|add:"-1" }}</span></td>
-                    <td>{{ feature.strand }}</td>
-                    <td>{% if feature.closest_gene_ensembl_id %}<a href="/search/feature/ensembl/{{ feature.closest_gene_ensembl_id }}">{{ feature.closest_gene_name }} ({{ feature.closest_gene_ensembl_id }})</a>{% else %}N/A{% endif %}</td>
+                    <td><span>{{ feature.chrom_name }}</span>: <span>{{ feature.location.lower }}-{{ feature.location.upper|add:"-1" }}:{{ feature.strand|if_strand }}</span></td>
+                    <td>{% if feature.closest_gene_ensembl_id %}<a href="/search/feature/ensembl/{{ feature.closest_gene_ensembl_id }}">{{ feature.closest_gene_ensembl_id }}</a>{% else %}{% endif %}</td>
                     <td>{{ feature.ref_genome }}.{{ feature.ref_genome_patch|default:"0" }}</td>
+                    <td>{{ feature.parent.name }}</td>
+                    <td><a href="{% url 'search:dna_features' 'accession' feature.accession_id %}">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="svg-link-arrow bi bi-arrow-up-right-square-fill" viewBox="0 0 16 16">
+                            <path fill-rule="evenodd" d="M14 0a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2zM5.904 10.803 10 6.707v2.768a.5.5 0 0 0 1 0V5.5a.5.5 0 0 0-.5-.5H6.525a.5.5 0 1 0 0 1h2.768l-4.096 4.096a.5.5 0 0 0 .707.707z"/>
+                        </svg>
+                        </a>
+                    </td>
                 </tr>
             {% endfor %}
         </table>
+
+        <div class="pagination">
+            <span class="step-links">
+                {% if features.has_previous %}
+                    <a href="?page=1" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page=1" hx-target="#features-table">« first</a>
+                    <a href="?page={{ features.previous_page_number }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.previous_page_number }}" hx-target="#features-table">previous</a>
+                {% endif %}
+
+                <span class="current">
+                    Page {{ features.number }} of {{ features.paginator.num_pages }}
+                </span>
+
+                {% if features.has_next %}
+                    <a href="?page={{ features.next_page_number }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.next_page_number }}" hx-target="#features-table">next</a>
+                    <a href="?page={{ features.paginator.num_pages }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.paginator.num_pages }}" hx-target="#features-table">last &raquo;</a>
```

Comment:
Pagination doesn't use "dist", which works on the dna_features.html page, but when paging through "Find Nearby DNA Features" on the dna_feature.html page the results won't be correct because they will only be for loc, not loc +/- dist.

Reply:
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/b4cb5367-8741-406b-a59f-e222694eecb1)
Looks like it may be working here

Reply:
That's not a "paged" request.

Reply:
Should be fixed now. Paging works with parameters and dist options